Azure WAF/Playbook - WAF Sentinel Playbook Block IP - New/README.md

```diff
@@ -0,0 +1,24 @@
+# Sentinel Playbook - Block IP - New
+
+[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fshabaz-github%2FAzure-Network-Security%2Fmaster%2FAzure%2520WAF%2FPlaybook%2520-%2520WAF%2520Sentinel%2520Playbook%2520Block%2520IP%2520-%2520New%2Ftemplate.json)
```

Comment:
@shabaz-github it appears from this link that the source file is still in your repo. Are you able to move the source file to our repo first?